src/App/Pages/Generator/GeneratorPage.xaml

```diff
@@ -445,7 +445,11 @@
                                 StyleClass="box-value"
                                 HorizontalOptions="End" 
                                 AutomationProperties.IsInAccessibleTree="True"
-                                AutomationProperties.Name="{u:I18n UppercaseAtoZ}"/>
+                                AutomationProperties.Name="{u:I18n UppercaseAtoZ}">
+                                <Switch.Effects>
+                                    <effects:HintColorSwitchEffect/>
+                                </Switch.Effects>
```

Comment:
If this effect needs to be applied to all switches on the app, make an implicit style defined globally and apply the effect to the style. If it's only for this page you can define the implicit style on this specific page so you don't set the effect on each control individually (unless there is some switch that doesn't require it)

Reply:
This effect only affects switches that have an hint set (it is set by the AutomationProperties.Name for the Android app). There is another switch that doesn't need this, on this page, as it doesn't have a hint.

Now, we could apply this to all switches anyway as it wouldn't have a negative impact and if in the future a new switch was added and had a hint it would have the correct color, even if it is unnecessary on some. What's your opinion?

Reply:
Removed the effect and added the content to the CustomSwitchRenderer that is applied globally